pytorch_binding/warp_rnnt/__init__.py

```diff
@@ -59,6 +89,11 @@ def rnnt_loss(log_probs: torch.FloatTensor,
         fastemit_lambda (float, optional): FastEmit regularization
             (https://arxiv.org/abs/2010.11148).
             Default: 0.0.
+        compact (bool, optional): Use compact layout, if True, shapes of inputs should be:
+            log_probs: (STU, 2)
```

Comment:
Could you clarify this shape (STU, 2) please. I guess it's similar to gather option? And please update `pytorch_binding/README.md`

Reply:
This is a mistake. Input should be (STU, V). The gather operation is conducted inside the forward computation. I'll update the readme later.